packages/fetchye/__tests__/computeKey.spec.js

```diff
@@ -58,4 +67,55 @@ describe('computeKey', () => {
     };
     expect(computeKey('uri', firstOptions).hash).toBe(computeKey('uri', secondOptions).hash);
   });
+
+  it('should return a different, stable hash, if the option mapKeyToCacheKey is passed', () => {
```

Comment:
This test cannot assert that the key is different as the spec states since it only generates one key. There is nothing to compare it to

Reply:
addressed in a693a07